public/Template/assets/mobirise/css/mbr-additional.css

```diff
@@ -148,17 +148,17 @@ body {
 }
 .btn-primary,
 .btn-primary:active {
-  background-color: #149dcc !important;
-  border-color: #149dcc !important;
+  background-color: yellowgreen !important;
```

Comment:
Please return this the way it is. There is a colored theme that is being used across the whole project.